### DISCUSSION TITLE
Intel QSV presets

### DISCUSSION THREAD

**Discussion by shahrulsong** [jlesage/docker-handbrake#297] [Q&A]
Hi, 

I saw that handbrake has a Hardware preset for Intel QSV such as H.265 QSV 2160p60 4K. However, in the main page of this repo, it says that it only supports H.264 for Intel QSV. Is there a preset for 4K QSV with H.264? I tried to just do with H.265 but the conversion failed.

**Comment by jlesage** [comment]
The QSV encoders that are shown depends on your CPU and on what support HandBrake detects.
Are you saying that you have a QSV encoder for H.265 but not H.264 ?
